stable/insights-agent/templates/aws-costs/cronjob.yaml

```diff
@@ -21,11 +30,11 @@ spec:
                 secretKeyRef:
                   name: {{ .Values.awscosts.secretName }}
                   key: AWS_ACCESS_KEY_ID
-            - name: AWS_ACCESS_KEY_ID
+            - name: AWS_ACCESS_KEY_PASSWORD
               valueFrom:
                 secretKeyRef:
                   name: {{ .Values.awscosts.secretName }}
-                  key: AWS_ACCESS_KEY_ID            
+                  key: AWS_ACCESS_KEY_PASSWORD            
```

Comment:
I do not think this will work... your keys in secret are  `AWS_SECRET_ACCESS_KEY` and `AWS_ACCESS_KEY_ID` there is  no `AWS_ACCESS_KEY_PASSWORD` so either you update here or the secret key

Reply:
Yea, I have added those in